wiki/Difficulties/en.md

```diff
@@ -11,13 +11,14 @@ tags:
 
 *Not to be confused with [Beatmaps](/wiki/Beatmaps).*
 
-A beatmap's **difficulty** describes the level of skill needed to complete it. The difficulty name and star rating of a beatmap usually give enough information to tell how difficult it is.
+A beatmap's **difficulty** describes the level of skill needed to complete the beatmap. The difficulty name and star rating of a beatmap usually give enough information to tell how difficult the beatmap is.
+![The star rating on this map](img/difficulty-example.jpg)
 
 ## Difficulty levels
 
 *See also: [Difficulty Naming](/wiki/Ranking_Criteria/Difficulty_Naming)*
 
-These are the typical levels of difficulty that a beatmap can fall under. Specific definitions and requirements of each can be found in their respective mode's [ranking criteria](/wiki/Ranking_Criteria).
+These are the most common beatmap difficulty names that can be found on almost every map. Specifics and definitions can be found under the specific mode's [ranking criteria](/wiki/Ranking_Criteria).
```

Comment:
Repetition again:
> *Specifics* and definitions (...) under the *specific* mode's

The introductory sentence could be also changed to something like
> The most common beatmap difficulty names for existing gameplay modes are listed below.